lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'send_text'
+class Order
+    attr_reader :order_list
+    def initialize
+        @order_list = []
```

Comment:
Could this name be a bit more descriptive? This name makes it sound like a list of orders.